nixpkgs_review/review.py

```diff
@@ -449,7 +441,30 @@ def list_packages(
             )
         tmp.flush()
         with open(tmp.name, encoding="utf-8") as f:
-            return parse_packages_xml(f)
+            return (system, parse_packages_xml(f))
+
+
+def list_packages(
+    nix_path: str,
+    systems: set[System],
+    allow: AllowedFeatures,
+    n_procs: int,
+    check_meta: bool = False,
+) -> dict[System, list[Package]]:
+    list_packages_partial = partial(
+        _list_packages_system,
+        nix_path=nix_path,
+        allow=allow,
+        check_meta=check_meta,
+    )
+
+    args: list[System] = list(systems)
+    with mp.Pool(n_procs) as pool:
```

Comment:
Can we change this to https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor
processes take longer to spawn than threads.

Reply:
Probably should also be changed in the other place.

Reply:
Done !